src/backend/utils/sanitize-html.js

```diff
@@ -8,38 +8,38 @@ module.exports = function(dirty) {
     // Add <img> to the list of allowed tags, see:
     // https://github.com/apostrophecms/sanitize-html#what-are-the-default-options
     allowedTags: [
-      'img',
-      'figure',
-      'br',
-      'p',
       'a',
-      'ol',
-      'nl',
-      'div',
-      'li',
-      'td',
-      'strong',
-      'pre',
-      'code',
       'b',
+      'blockquote',
+      'br',
+      'caption',
+      'code',
+      'div',
       'em',
-      'ul',
-      'tr',
+      'figure',
+      'h1',
+      'h2',
       'h3',
-      'td',
-      'i',
       'h4',
-      'pre',
-      'blockquote',
+      'h5',
+      'h6',
       'hr',
+      'i',
+      'img',
+      'li',
+      'nl',
```

Comment:
I'm not sure what `nl` is, seems like a previous typo.  Could you remove here, please?